src/net.cpp

```diff
@@ -1951,6 +1951,19 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
     }
 }
 
+std::vector<CAddress> CConnman::GetBlockRelayNodeAddresses() const
```

Comment:
This function name doesn't reflect that it's inbound-only

Reply:
Maybe it actually should be named "GetCurrentAnchors"

Reply:
> This function name doesn't reflect that it's inbound-only

You mean "outbound"?

Reply:
> Maybe it actually should be named "GetCurrentAnchors"

[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-653533271).